experiments/DemoToString/DemoToString.cpp

```diff
@@ -118,14 +118,6 @@ BENCHMARK(DemoToString, table, 128, 1000000)
 	celero::DoNotOptimizeAway(uint8str[x]);
 }
 
-BENCHMARK(DemoToString, itoa, 128, 1000000)
```

Comment:
I like this experiment, but understand the incompatibility issue.  Could this be accomplished with ```#ifdef(_ito)`` as opposed to deleting entire code block?